e2e/tests/cluster/cluster_test.go

```diff
@@ -39,6 +41,15 @@ func TestMain(m *testing.M) {
 	code := m.Run()
 	state.EndTime = time.Now()
 
+	// Notify if we receive any signal
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
```

Comment:
Looks like these signals have no impact. Is it just for logs?

Reply:
Yeah, just to register in logs if we received any signal, potentially from k8s during a node removal or something like that.